src/test/scala/dotty/communitybuild/CommunityBuildTest.scala

```diff
@@ -69,4 +69,9 @@ class CommunityBuildTest extends BuildTest {
     branch  = "dotty-library-2.13",
     command = "library/compile"
   )
+
+  @Test def pdbp = project(
```

Comment:
If you don't specify a branch, it will default to `dotty` which does not exist currently.

We also want to build a stable branch, so I would advise against using `master`. I usually fork the repo into `dotty-staging` to make sure I build something stable.